ohapi/command_line.py

```diff
@@ -237,9 +240,16 @@ def upload_metadata(directory, create_csv='', review='',
     set_log_level(debug, verbose)
 
     max_bytes = parse_size(max_size)
-
-    if create_csv:
+    if create_csv and review:
+        raise ValueError("Either create_csv must be true or review must be " +
+                         "true but not both")
+    if review:
+        review_metadata_csv(directory, review, max_bytes=max_bytes)
```

Comment:
This results in an error, as `review_metadata_csv` doesn't seem to take a `max_bytes` argument

Reply:
Also, after that issue is fixed – this isn't returning any response that indicates whether the resulting review was good or false. (In the case of a valid file, it's not rendering any output at all.)